README.md

```diff
@@ -2,6 +2,18 @@
 
 A java integration library for AeroGear Digger
 
+## Project Info
+
+|                 | Project Info  |
+| --------------- | ------------- |
+| License:        | Apache License, Version 2.0  |
+| Build:          | Maven  |
+| Documentation:  | tbd  |
```

Comment:
For documentation we can reference "https://github.com/aerogear/digger-jenkins" repo as this is just client for this general solution. WDYT?